tests/bluetooth/tester/src/audio/btp_vcp.c

```diff
@@ -495,31 +537,18 @@ static void set_register_params(uint8_t gain_mode)
 		vcp_register_param.aics_param[i].cb = &aics_server_cb;
 	}
 
-	vcp_register_param.step = 1;
-	vcp_register_param.mute = BT_VCP_STATE_UNMUTED;
-	vcp_register_param.volume = 100;
+	vcp_register_param.step = step;
+	if (mute) {
+		vcp_register_param.mute = BT_VCP_STATE_MUTED;
+	} else {
+		vcp_register_param.mute = BT_VCP_STATE_UNMUTED;
+	}
```

Comment:
`mute` should either be a `bool`, or we should just do

```suggestion
	vcp_register_param.mute = mute;
```